src/common/layout/ContentPage.js

```diff
@@ -54,7 +54,7 @@ class ContentPage extends React.Component<Props> {
             height: calc(100% - (64px));
           }
           .FAQ {
-            width: 480px;
+            width: 650px;
```

Comment:
the `650px` size should be **only** when logged in according to designs, when logged out it should stay as `480px`